msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/mwpj/TestCase2521950.kt

```diff
@@ -0,0 +1,146 @@
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.client.msal.automationapp.testpass.broker.mwpj
+
+import com.microsoft.identity.client.msal.automationapp.R
+import com.microsoft.identity.client.msal.automationapp.testpass.broker.AbstractMsalBrokerTest
+import com.microsoft.identity.client.ui.automation.annotations.LocalBrokerHostDebugUiTest
+import com.microsoft.identity.client.ui.automation.annotations.SupportedBrokers
+import com.microsoft.identity.client.ui.automation.broker.BrokerHost
+import com.microsoft.identity.client.ui.automation.rules.LoadLabUserTestRule
+import com.microsoft.identity.labapi.utilities.client.ILabAccount
+import com.microsoft.identity.labapi.utilities.client.LabQuery
+import com.microsoft.identity.labapi.utilities.constants.AzureEnvironment
+import com.microsoft.identity.labapi.utilities.constants.TempUserType
+import com.microsoft.identity.labapi.utilities.constants.UserType
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TestRule
+
+// https://identitydivision.visualstudio.com/Engineering/_workitems/edit/2521950
+@LocalBrokerHostDebugUiTest
+@SupportedBrokers(brokers = [BrokerHost::class])
+class TestCase2521950 : AbstractMsalBrokerTest() {
+    private lateinit var mUsGovLabAccount: ILabAccount
+    private lateinit var mTempAccount: ILabAccount
+
+    @get:Rule
+    val loadUsGovLabAccountUserRule: TestRule = LoadLabUserTestRule(getAdditionalLabQuery())
+
+    @get:Rule
+    val loadAdditionalLabUserRule: TestRule = LoadLabUserTestRule(TempUserType.BASIC)
+
+    @Test
+    fun test_2521950() {
+
+        val username = mLabAccount.username
+        val password = mLabAccount.password
+        val usernameTmp = mTempAccount.username
+        val passwordTmp = mTempAccount.password
+        val usGovUsername = mUsGovLabAccount.username
+        val usGovPassword = mUsGovLabAccount.password
+        val brokerHostApp = broker as BrokerHost
+        brokerHostApp.enableMultipleWpj()
```

Comment:
If we are going to add a @before method, we might as well enable MWPJ in there. Although I think we can do without the @before method, so maybe this is fine if we get rid of that method

Reply:
done 79ed44ed0cb3060fe9ad4fcecdfb5c289d55e546